Library/Formula/mariadb-connector-c.rb

```diff
@@ -0,0 +1,54 @@
+class MariadbClientRequirement < Requirement
+  fatal true
+  default_formula "mariadb"
+  satisfy { which "mysql" }
+end
+
+class MariadbConnectorC < Formula
+  desc "MySQL & MariaDB database connector for C/C++ applications"
+  homepage "https://github.com/MariaDB/mariadb-connector-c"
+  stable do
+    url "https://github.com/MariaDB/mariadb-connector-c.git",
+        :revision => "330b7fbdbabad44180862316d8925b302d1a8409"
+    version "3.0.0"
+    # link _libiconv_* symbols on OS X
+    # https://github.com/MariaDB/mariadb-connector-c/issues/4
+    patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!